edge-apps/tfl-bus-status/static/js/script.js

```diff
@@ -227,8 +228,11 @@ async function fetchBusData () {
     busArrival.innerHTML = error
     document.querySelector('.bus-list').classList.add('hidden')
   }
-  // Send signal to load the screen once content is ready.
-  screenly.signalReadyForRendering()
+
+  // Send signal to load the screen once content is ready, if the function exists.
+  if (typeof screenly !== 'undefined' && typeof screenly.signalReadyForRendering === 'function') {
```

Comment:
why are those checks still needed? We discussed we should just not report to sentry.

Reply:
I'm sorry, but do you know how we plan to add the sentry?